src/csharp/Extensions/Microsoft.Spark.Extensions.DotNet.Interactive.UnitTest/PackageHelperTests.cs

```diff
@@ -0,0 +1,98 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.DotNet.Interactive.Utility;
+using Microsoft.Spark.UnitTest.TestUtils;
+using Microsoft.Spark.Utils;
+using Moq;
+using Xunit;
+
+namespace Microsoft.Spark.Extensions.DotNet.Interactive.UnitTest
+{
+    public class PackageHelperTests
+    {
+        [Fact]
+        public void TestPackageHelper()
+        {
+            using var tempDir = new TemporaryDirectory();
+
+            string packageName = "package.name";
+            string packageVersion = "0.1.0";
+            string packageRootPath =
+                Path.Combine(tempDir.Path, "path", "to", "packages", packageName, packageVersion);
+            string packageFrameworkPath = Path.Combine(packageRootPath, "lib", "framework");
+
+            Directory.CreateDirectory(packageRootPath);
+            FileInfo nugetFile = new FileInfo(
+                Path.Combine(packageRootPath, $"{packageName}.{packageVersion}.nupkg"));
+            using (File.Create(nugetFile.FullName))
+            {
+            }
+            
+            IReadOnlyList<FileInfo> assemblyPaths = new List<FileInfo>
+            {
+                new FileInfo(Path.Combine(packageFrameworkPath, "1.dll")),
+                new FileInfo(Path.Combine(packageFrameworkPath, "2.dll"))
+            }.AsReadOnly();
```

Comment:
Is `AsReadOnly()` needed? If not, you can just do `var assemblyPaths = ...` and just pass it to `ResolvedPackageReference` constructor.